applications/MappingApplication/mpi_extension/custom_python/mapping_mpi_extension.cpp

```diff
@@ -21,15 +21,15 @@
 #include "custom_mappers/nearest_element_mapper.h"
 #include "custom_utilities/mapper_mpi_define.h"
 #include "custom_utilities/mapper_mpi_backend.h"
-#include "custom_utilities/mapper_factory.h"
-#include "custom_python/add_mapper_to_python.h"
+#include "factories/mapper_factory.h"
+#include "python/add_mapper_to_python.h"
 
 namespace Kratos {
 namespace Python {
 
 PYBIND11_MODULE(KratosMappingMPIExtension,m)
 {
-    AddMapperToPython<MPIMapperDefinitions::SparseSpaceType, MPIMapperDefinitions::DenseSpaceType>(m);
+    AddMappingToPython<MPIMapperDefinitions::SparseSpaceType, MPIMapperDefinitions::DenseSpaceType>(m);
```

Comment:
Not this line in particular but, why is the mapper interface moved to the core and the mpi extension is still in the application? shouldn't we move the interface of the extension to `KratosMPICore` as well?

Reply:
unfortunately the MPI extension still depends on Trilinos which is why I cannot move it to the Core
I am currently refactoring with the Kratos-CSR matrices, once I am done with that I will move it as you suggest